### PR TITLE
[RN Mobile] Port quote block to the ReactNative

### PR DESCRIPTION
**Pull request by daniloercoli**
## Description
This PR contains the initial set of required changes for ReactNative mobile app be able to use the quote block. Changes can be tested using this [PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/214).

## How has this been tested?

- [ ]  Make sure the (web) GB build and webapp work normally
- [ ]  Make sure the GB mobile app works normally and quote block is available in the editor.

### CODE REVIEW

File: packages/block-library/src/quote/edit.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import {
+	BlockControls,
+	AlignmentToolbar,
+	RichText,
+} from '@wordpress/editor';
+
+export default function edit( { attributes, setAttributes, isSelected, mergeBlocks, onReplace, className } ) {
```
Comment (gziolo) [inline packages/block-library/src/quote/edit.js]: We use a name starting with upper case for the component names. In other places we use something which would translate to `QuoteEdit` here.

File: packages/block-library/src/quote/edit.native.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+						onContentSizeChange={ ( event ) => {
+							setAttributes( {
+								...this.props.attributes,
+								aztecHeight: event.aztecHeight,
+							} );
+						}
+						}
+						/* translators: the individual or entity quoted */
+						placeholder={ placeholder || __( 'Write citation…' ) }
+						className="wp-block-quote__citation"
```
Comment (gziolo) [inline packages/block-library/src/quote/edit.native.js]: I don't think you want to preserve class name for RN part.

**Review by gziolo — COMMENTED** [review]
Web part looks good. There is one minor thing to fix. I didn't look very closely at the mobile part. I would appreciate another 👀 on that file.

Oh, there is merge conflict to be resolved, which probably means that `edit` function has changed.

**Comment by hypest** [comment]
Does it make sense to update this PR to bring it up to date or should we drop it for a new one @daniloercoli ?

**Comment by daniloercoli** [comment]
> Does it make sense to update this PR to bring it up to date or should we drop it for a new one @daniloercoli ?

I guess it's easier to start from scratch. Will close this one.

**Pull request closed without merging**